src/domain/valueObject/password.go

```diff
@@ -28,6 +26,7 @@ func NewPassword(value interface{}) (password Password, err error) {
 	return Password(stringValue), nil
 }
 
+// TODO: remove this constructor when no longer used.
```

Comment:
Same as [this comment](https://github.com/speedianet/os/pull/142/files#r1683023375).

Reply:
And same for what I mentioned above, hehe. Won't be repeating, so please remove this TODOs lines.

Reply:
Done.